Formula/dashing.rb

```diff
@@ -0,0 +1,36 @@
+class Dashing < Formula
+  desc "Generate Dash documentation from HTML files"
+  homepage "https://github.com/technosophos/dashing"
+  url "https://github.com/technosophos/dashing/archive/0.3.0.tar.gz"
+  sha256 "f6569f3df80c964c0482e7adc1450ea44532d8da887091d099ce42a908fc8136"
+
+  depends_on "glide" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+
+    (buildpath/"src/github.com/technosophos/dashing").install buildpath.children
+    cd "src/github.com/technosophos/dashing" do
+      system "glide", "install"
+      system "go", "build", "-o", bin/"peco", "dashing.go"
```

Comment:
also need to pass -ldflags "-X main.version=#{version}" dashing.go

Reply:
@ilovezfs Ok, I implemented that.

Reply:
Hmm, I didn't realize, but `brew audit` doesn't like what I did by putting the whole command on one line. Unfortunately, `go build` doesn't like the way the `brew audit` wants me to do it. What am I supposed to do?